app/src/main/java/io/homeassistant/companion/android/launch/LaunchActivity.kt

```diff
@@ -81,7 +84,20 @@ class LaunchActivity : AppCompatActivity(), LaunchView {
     override fun displayWebview() {
         presenter.setSessionExpireMillis(0)
 
-        startActivity(WebViewActivity.newInstance(this, intent.data?.path))
+        val isAutomotive = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            this.packageManager.hasSystemFeature(PackageManager.FEATURE_AUTOMOTIVE)
+        } else {
+            false
+        }
```

Comment:
Why do we need a version check + `this` here? See for example `BaseActivity` where it's much simpler.